src/components/dashboard/AdmissionsDash.jsx

```diff
@@ -88,16 +76,30 @@ const AdmissionsDash = (props) => {
     numberOfRows,
     page,
   } = useSelector((state) => state.students);
-  // console.log(
-  //   filterColumns,
-  //   url,
-  //   studentData,
-  //   fromDate,
-  //   toDate,
-  //   totalData,
-  //   stage,
-  //   page
-  // );
+  const [allSchools, setAllSchools] = React.useState()
+
+  useEffect(() => {
+    axios.get(`${baseURL}school`).then((res) => {
+      setAllSchools(res.data)
+    }).catch((err) => {
+      console.log('err', err)
+    })
+  },[])
+
+  let allSchoolOptions = [
+    {
+      value: "default",
+      label: "All",
+    },
+  ]
+  allSchools?.forEach((item, index) => {
+    let obj =  {
+      value: index,
+      label: item.name,
+    }
+    allSchoolOptions.push(obj)
+  });
```

Comment:
Need to use `index + 1` here instead of `index` since the first school has an id of `1`, not `0`. But for consistency with the backend (in case ids change there), I'd suggest using the item's id instead.
```suggestion
  const allSchoolOptions = [
    {
      value: 'default',
      label: 'All',
    },
  ].concat(
    allSchools.map((item) => ({
      value: item.id,
      label: item.name,
    }))
  );
```